tools/linter/codespell/.codespell.skip

```diff
@@ -8,6 +8,8 @@
 *.jpg
 *.ico
 *.svg
+*.js
```

Comment:
maybe skip the generated `docs/html/*` instead ? so codespell only runs against the src